src/styles/utils/_variables.scss

```
@@ -0,0 +1,7 @@
+$block_colors :
+  "1" rgb(255,0,0),
```

you can use percentages with rgb
```suggestion
  "1" rgb(100%,0,0),
```